### ISSUE TITLE
markdown perview  is sometimes notwork

### ISSUE THREAD

**Issue by wjk930726** [Microsoft/vscode#24098]
- VSCode Version: Code 1.11.1 (d9484d12b38879b7f4cdd1150efeb2fd2c1fbf39, 2017-04-06T13:58:12.954Z)
- OS Version: Windows_NT ia32 10.0.14393
- Extensions: 

|Extension|Author|Version|
|---|---|---|
|theme-seti|bialikover|0.0.1|
|beautify|HookyQR|1.0.2|
|cpptools|ms-vscode|0.10.5|
|Theme-MarkdownKit|ms-vscode|0.1.4|
|seti-icons|qinjia|0.1.3|;

---

Steps to Reproduce:

1.split screen
2.one side write markdown, the other side real-time preview
3.even save markdown file,the other side did not show the correct preview

![2017-04-07](https://cloud.githubusercontent.com/assets/21329621/24765781/ba23ba38-1b2b-11e7-805e-0a6df667ea84.png)

**Comment by chrmarti** [comment]
Can you check if there is anything in the devtools' console (Help > Toggle Developer Tools)?

**Comment by wjk930726** [comment]
only one warning'KeyboardEvent.keyIdentifier' is deprecated and will be removed in M54, around October 2016. See https://www.chromestatus.com/features/5316065118650368 for more details.
which is in js:103 'KeyboardEvent.keyIdentifier' is deprecated and will be removed in M54, around October 

<img width="1280" alt="2017-04-07 9 32 15" src="https://cloud.githubusercontent.com/assets/21329621/24781935/98c1d768-1b75-11e7-8dab-853fc4b1b302.png">

But I found that I just set  language mode for the markdown, it can correctly preview.
